config/crs/kubernetes/pipelines-as-code/operator_v1alpha1_addon_cr.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonPipelinesAsCode
+metadata:
+  name: pac
+spec:
+  targetNamespace: pipelines-as-code
```

Comment:
We have same yaml under `crs/kubernetes` and `crs/openshift` if nothing is different should we keep in common place?

Reply:
currently for all components we do that way 
lets address it separately